docs/source/subgraphs.md

```diff
@@ -180,9 +180,12 @@ Alternatively, you can provide `-`, in which case the command uses an SDL string
 
 <td>
 
-**Required.** Used by a gateway running in [managed federation mode](https://www.apollographql.com/docs/federation/managed-federation/overview/).
+**Optional.** Used by a gateway running in [managed federation mode](https://www.apollographql.com/docs/federation/managed-federation/overview/).
 
-If you're running a subgraph that hasn't been deployed yet or isn't using managed federation, you can pass a placeholder URL or leave the flag empty.
+The first time you publish a subgraph, this argument is required (though you can set the value to an empty string if your subgraph has not yet been deployed, or if you aren't using managed federation).
+
+After your first publish, this argument is no longer required, and your graph router will continue to route requests
+for this subgraph to the routing URL it recorded from your first publish.
```

Comment:
this is... a bit wordy. i wanted to shy away from using the term "upsert" here since it's a bit jargony and expects people to know what it means. perhaps this could be a bit more succinct though

cc @StephenBarlow any thoughts here?

Reply:
It's nearly there! Suggested edit:

The URL that your gateway uses to communicate with the subgraph in a [managed federation architecture](link).

**Required** the first time you publish a particular subgraph. Provide the empty string if your subgraph isn't deployed yet, or if you aren't using managed federation.

**Optional** after your first publish. Provide only if you need to change the subgraph's routing URL.